Bottom: 084430c58f64bc1518e2104e331686e48dad2e40
Top:    7c62e953e8504807e1cad035ff36a6bf351f8b46
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:46:08 +0000

Refresh of hid-step-add-copper-layer

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0723343..e2f471a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -63,8 +63,8 @@
 
 
 #define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
-#define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
+#define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 
 static GList *object3d_test_objects = NULL;
